src/test/java/net/snowflake/ingest/streaming/internal/datatypes/IcebergStringIT.java

```diff
@@ -84,15 +89,18 @@ public void testStringAndQueries() throws Exception {
     testIcebergIngestAndQuery(
         "string", Arrays.asList(null, null, null, null, "aaa"),
         "select COUNT(*) from {tableName} where {columnName} is null", Arrays.asList(4L));
-    testIcebergIngestAndQuery(
-        "string",
-        Arrays.asList(StringUtils.repeat("a", 16 * 1024 * 1024), null, null, null, "aaa"),
-        "select MAX({columnName}) from {tableName}",
-        Arrays.asList(StringUtils.repeat("a", 16 * 1024 * 1024)));
     testIcebergIngestAndQuery(
         "string",
         Arrays.asList(StringUtils.repeat("a", 33), StringUtils.repeat("*", 3), null, ""),
         "select MAX(LENGTH({columnName})) from {tableName}",
         Arrays.asList(33L));
+    // TODO: Add this back after the following JIRA is fixed
```

Comment:
Lets keep the test in with the 16MB-1 hack that you have so it doesn't regress further?

Reply:
Added back